package.json

```diff
@@ -30,7 +30,8 @@
         "docs": "shx rm -rf docs && NODE_OPTIONS=--max_old_space_size=16000 typedoc && shx cp ./{.nojekyll,wallets.png} docs/",
         "deploy:docs": "pnpm run docs && gh-pages --dist docs --dotfiles",
         "example": "pnpm run --filter {packages/starter/example} export",
-        "deploy:example": "pnpm run example && gh-pages --dist packages/starter/example/out --dest example --dotfiles"
+        "deploy:example": "pnpm run example && gh-pages --dist packages/starter/example/out --dest example --dotfiles",
+        "tsc": "tsc --build --verbose tsconfig.all.json"
```

Comment:
Added this as ReadMe mentions on step 3 to `pnpm run tsc` to build packages, but when running it gives out saying `Missing Script Error`

Reply:
Ah, this is because the readme should be updated instead to call `pnpm run build:ts`.

Reply:
Cool, i will revert this change then!